bylaws.md

```diff
@@ -1,58 +1,111 @@
-5. All Active Members shall be notified of a special meeting and its purpose by the Secretary one week prior to the meeting. 
-6. No business shall be conducted at a called special meeting except that for which the meeting was called. 
+1. The Chapter shall charge a fee to each new member which shall consist of a Chapter Initiation Fee in the amount of twenty dollars in addition to an Association Initiation Fee which shall be determined by the National Association.
+2. Each initiate shall be required to pay Chapter and Association initiation fees prior to his/her initiation.
+3. Special assessments may be made at any time by a three-fourths affirmative vote of the Active Members of the Chapter.
 
-## Article IV: Special Duties 
+## Article VI: Induction of New Members
 
-1. In addition to his/her regular duties, the Secretary shall make all arrangements for the regular and special meetings.
+### Section I: Induction Ceremony
 
-## Article V: Finances 
+1. The induction of new Members shall be held within eight weeks of the time of their election.
+2. The induction shall specifically avoid any practice which conflicts with WPI regulations or which constitutes an attack upon the personal dignity of the initiates. The activities shall be devised to stimulate the inductees’ interest in Computer Science and to acquaint them better with the Members of the Chapter.
 
-1. The initiation fee charged each new Member by the Chapter shall be twenty dollars, in addition to the initiation fee of the Association.
-2. Each initiate shall be required to pay all dues and initiation fees prior to his/her initiation.
-3. Special assessments may be made at any time, upon approval of three-fourths of the Active Members of the Chapter.
+### Section II: Eligibility of Candidates
 
-## Article VI: Induction Ceremony 
+The eligibility of students, former students, and faculty for election to membership in this Chapter of the Upsilon Pi Epsilon Association shall be determined as follows:
 
-1. The induction of new Members shall be held within four weeks of the time of their election.
-2. The induction shall specifically avoid any practice which conflicts with WPI regulations or which constitutes an attack upon the personal dignity for the initiates. The activities shall be devised to stimulate the inductees’ interest in Computer Science and to acquaint them better with the Members of the Chapter.
+1. A graduate student must be a candidate for a graduate degree in Computer Science, and must have completed at least twelve credits of graduate study, three-quarters of this with a grade level of A.
+2. Undergraduate student eligibility shall require junior or senior status, a minimum of two Computer Science related units, with greater than fifty percent of A's in Computer Science, and a majority of A's and B's in non-Computer Science courses.
```

Comment:
Shouldn't this be "45 Semester Hours/ a year and a half minimum instead of junior/senior status". This seems to contradict the email that I received about what you all discussed during the meeting.